docs/user/reference/pings/index.md

```diff
@@ -1,6 +1,9 @@
-# Submitting a custom ping
+# Pings
 
-To collect and queue a custom ping for eventual uploading, call the `submit` method on the `PingType` object that the Glean SDK generated for your ping.
+## Submission API
+
+### `submit`
+Collect and queue a custom ping for eventual uploading.
```

Comment:
For the sake of redundancy, can we add an info or warning box saying that internal pings don't need to be submitted? Something like "HEY, THIS REALLY ONLY MATTERS IF YOU HAVE CUSTOM PINGS!!!! GO AWAY OTHERWISE!!11!!!", but nicer and then link to the internal pings page, ofc.